### DISCUSSION TITLE
Blazor as a Office addin?

### DISCUSSION THREAD

**Discussion by sturlath** [dotnet/aspnetcore#26419] [General] [answered]
There is an [article from nov 2019](https://developer.microsoft.com/en-us/office/blogs/office-add-ins-community-call-november-13-2019/) that says

> Will Blazor be supported for Office Add-ins?

> No, it will be a React Office.js add-in. We don’t have any plans to support Blazor yet. 

I have searched and [tweeted](https://twitter.com/SturlaThorvalds/status/1309834124446896128) (sadly with no response) but I can´t find information if this is still the case or if with .net 5 this is now just a posibility?

I find it really strange if this programming model (C# over js) will not be available someday, but I would like to know when.

So anybody got **any** information on this for me?

**Comment by SteveSandersonMS** [comment, marked as answer]
For information about the Office Add-In programming model, it would be best to post in a support channel for Office, as you'll get more precise information. In terms of .NET 5, I'm not aware that it changes anything as far as what Office Add-Ins support. Sorry about that!

**Reply by sturlath** [reply]
For future interested parties, I found the correct repo (I think) and asked [the question there](https://github.com/OfficeDev/office-js/issues/1418). Let's see what comes out of that :-)
